package.json

```diff
@@ -96,7 +96,7 @@
     "resolve-from": "^2.0.0",
     "rimraf": "^2.5.4",
     "signal-exit": "^3.0.1",
-    "spawn-wrap": "^1.4.0",
+    "spawn-wrap": "1.3.8",
```

Comment:
i've noticed some weirdness around pinning versions in npm 5; using `=1.3.8` seems to avoid it

Reply:
Is there a reason `spawn-wrap` was downgraded? There is a fix in version `1.4.0` that can be useful for people that want to run nyc in a container.